easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.33.0-intel-2020b.eb

```diff
@@ -0,0 +1,52 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'NanoPlot'
+version = '1.33.0'
+
+homepage = 'https://github.com/wdecoster/NanoPlot'
+description = "Plotting suite for long read sequencing data and alignments"
+
+toolchain = {'name': 'intel', 'version': '2020b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['caf69a5789625e746814238cb4f0d510ed6768a094566e8f1dcaa77933c4d6fd']
+
+# statsmodels and pauvre are locked to specific versions, although newer versions seems to work fine.
+# replace '==' with '>=' which works fine with current latest versions
+preinstallopts = "sed -i'' 's/==/>=/g' setup.py && "
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Biopython', '1.78'),
+    ('Pysam', '0.16.0.1'),
+    ('nanomath', '1.2.0'),
+    ('nanoget', '1.15.0'),
+    ('Seaborn', '0.11.1'),
+    ('plotly.py', '4.14.3'),
+    ('pauvre', '0.1924'),
+    ('statsmodels', '0.12.1'),
+    ('Arrow', '0.17.1'),  # for pyarrow
+    ('Kaleido', '0.1.0'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+# replace strict Seaborn & pauvre version requirement with minimal version requirements
+preinstallopts = "sed -i 's/==0/>=0/g' setup.py && "
```

Comment:
`preinstallopts` is redefined here, seems a leftover from the easyconfig of `NanoComp`

Reply:
fixed in f999a60b53